packages/roosterjs-editor-api/lib/format/getFormatState.ts

```diff
@@ -45,6 +45,10 @@ function queryCommandState(editor: Editor, command: string): boolean {
 /**
  * Get format state at cursor
```

Comment:
Please add a brief introduction what is format state.

Reply:
Added